### DISCUSSION TITLE
Document passing arguments to dream2nix

### DISCUSSION THREAD

**Discussion by mightyiam** [90-008/nix-cargo-integration#173] [Ideas]
As follow-up to https://github.com/yusdacra/nix-cargo-integration/discussions/157#discussioncomment-10037474, I think that the means for passing arguments to dream2nix better be documented. I couldn't find such documentation by searching https://flake.parts/options/nix-cargo-integration for `dream2nix`.

**Comment by 90-008** [comment]
i thought i already mentioned this in the `drvConfig` and `depsDrvConfig` docs but i apparently didn't, so i added a line clarifying it just now, should be up whenever flake.parts updates

**Reply by mightyiam** [reply]
:pray:
